src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-img {
+  margin: 10px 50px;
+}
+
+.nav {
+  height: 60px;
+  text-transform: uppercase;
+}
+
+.nav_list {
+  list-style-type: none;
+  margin: 0;
+  margin-right: 50px;
+  padding: 0;
+  display: flex;
```

Comment:
https://github.com/mate-academy/layout_moyo-header/pull/3238/files#r1211482979

Reply:
Sorry I do not understand this comment. Please can you explain me.